terraform/deployments/govuk-publishing-platform/signon_bearer_tokens.tf

```diff
@@ -31,140 +31,74 @@ locals {
-  workspace                       = local.workspace
-}
-
-module "draft_content_store_to_publishing_api_bearer_token" {
-  source = "../../modules/signon_bearer_token"
-
-  additional_tags                 = local.additional_tags
-  api_user_email                  = local.signon_api_user.draft_content_store
-  app_name                        = local.signon_app.publishing_api.name
-  environment                     = var.govuk_environment
-  name                            = "dcs_to_pub_api"
-  private_subnets                 = local.private_subnets
-  signon_admin_password_arn       = aws_secretsmanager_secret.signon_admin_password.arn
-  signon_host                     = module.signon.virtual_service_name
-  signon_lambda_security_group_id = aws_security_group.signon_lambda.id
-  workspace                       = local.workspace
-}
-
-module "content_store_to_router_api_bearer_token" {
-  source = "../../modules/signon_bearer_token"
+  signon_bearer_tokens = {
+    cs_to_pub_api = {
+      api_user = local.signon_api_user.content_store
+      app      = local.signon_app.publishing_api.name
+    }
+
+    cs_to_router_api = {
+      api_user = local.signon_api_user.content_store
+      app      = local.signon_app.router_api.name
+    }
+
+    dcs_to_pub_api = {
+      api_user = local.signon_api_user.draft_content_store
+      app      = local.signon_app.publishing_api.name
+    }
+
+    dcs_to_draft_router_api = {
+      api_user = local.signon_api_user.draft_content_store
+      app      = local.signon_app.draft_router_api.name
+    }
+
+    pub_to_pub_api = {
+      api_user = local.signon_api_user.publisher
+      app      = local.signon_app.publishing_api.name
+    }
+
+    pub_api_to_cs = {
+      api_user = local.signon_api_user.publishing_api
+      app      = local.signon_app.content_store.name
+    }
+
+    pub_api_to_dcs = {
+      api_user = local.signon_api_user.publishing_api
+      app      = local.signon_app.draft_content_store.name
+    }
+
+    pub_api_to_router_api = {
+      api_user = local.signon_api_user.publishing_api
+      app      = local.signon_app.router_api.name
+    }
+
+    frontend_to_pub_api = {
+      api_user = local.signon_api_user.frontend
+      app      = local.signon_app.publishing_api.name
+    }
+  }
 
-  additional_tags                 = local.additional_tags
-  api_user_email                  = local.signon_api_user.content_store
-  app_name                        = local.signon_app.router_api.name
-  environment                     = var.govuk_environment
-  name                            = "cs_to_router_api"
-  private_subnets                 = local.private_subnets
-  signon_admin_password_arn       = aws_secretsmanager_secret.signon_admin_password.arn
-  signon_host                     = module.signon.virtual_service_name
-  signon_lambda_security_group_id = aws_security_group.signon_lambda.id
-  workspace                       = local.workspace
+  generated_bearer_tokens = {
```

Comment:
To check I've understood this - this is bearer tokens by API user, right?

Reply:
Yeah exactly